backend/backend.go

```diff
@@ -55,5 +55,11 @@ func (b *Backend) Run() error {
 		})
 	})
 
+	r.GET("/dnsapi/getAllDomainMetadata/:name", func(c *gin.Context) {
+		c.JSON(200, gin.H{
+			"result": false,
```

Comment:
Thank you for your contribution!

The documentation at https://doc.powerdns.com/authoritative/backends/remote.html#getalldomainmetadata states that `You must always return something, if there are no values, you shall return empty set.` so that instead of `false` we should return an empty JSON object (probably via `gin.H{}`).

Reply:
very thanks